### PR TITLE
internal/provisioner: Use local variable  to replace the long access chain of fields

### PR DESCRIPTION
**Pull request by izturn**
it seems the fields of object nest too deeply, so use  a local variable to replace it to make read code easy

Signed-off-by: Gang Liu <gang.liu@daocloud.io>

### CODE REVIEW

**Review by youngnick — COMMENTED** [review]
The change itself looks fine to me, could you please run `gofmt -s` and check that `make lint` succeeds?

**Comment by izturn** [comment]
thx @youngnick  done, but how can i set the release note label?

File: test/e2e/deployment.go [hunk 1]
```diff
@@ -339,9 +339,6 @@ func (d *Deployment) WaitForContourDeploymentUpdated() error {
 		if err := d.client.List(context.TODO(), pods, labelSelectAppContour); err != nil {
 			return false, err
 		}
-		if pods == nil {
```
Comment (tsaarni) [inline test/e2e/deployment.go:342]: I wonder what was the reason for removing this?

**Review by youngnick — APPROVED** [review]
LGTM, thanks @izturn!

**Pull request merged**